lib/neo4j/shared/property.rb

```diff
@@ -17,15 +17,14 @@ class MultiparameterAssignmentError < StandardError; end
 
     # TODO: Remove the commented :super entirely once this code is part of a release.
     # It calls an init method in active_attr that has a very negative impact on performance.
-    def initialize(attributes = {}, _options = nil)
-      attributes = process_attributes(attributes) unless attributes.empty?
+    def initialize(attributes = nil, _options = nil)
```

Comment:
Why change this to `nil`?  I could see changing the `_options` to default to `{}` (also it seems like `_options` isn't used...)